db/seeds.rb

```diff
@@ -91,7 +91,6 @@
 user4.likes.create!(post: post5)
 user2.likes.create!(post: post5)
 user2.likes.create!(post: post6)
-user2.likes.create!(post: post6)
```

Comment:
- [ ] Make sure to add the user email and password to your user seeds.rb  line 18 up to line 22 to prevent this error message 👇🏻 

![Screenshot from 2022-12-17 22-12-15](https://user-images.githubusercontent.com/79058364/208264478-06be04e9-1f84-4ab6-8759-09540917453b.png)